src/txgraph.cpp

```diff
@@ -1942,6 +1957,13 @@ void TxGraphImpl::SanityCheck() const
     }
 }
 
+void TxGraphImpl::DoWork() noexcept
```

Comment:
As it is right now, it's unused in the main PR.

I've been thinking when `DoWork` can be called in the mempool lifecycle and whether that would make a difference because everything is done lazily and we only apply staged changes when we want to get the current state. (specifically a scenario where after some lazy additions it cheap to just call `DoWork` and not wait until we want to get a state.)


Maybe after trimming due to a reorg, after multiple additions of transactions e.g loading new mempool.